data/human/deep missions.txt

```diff
@@ -2286,6 +2287,7 @@ mission "Deep: Scientist Rescue 3A"
 		set "license: City-Ship"
 		event "battle in arneb end"
 		log `Destroyed the pirate Bactrian that was attempting to prevent the rescue of the Star Queen and was given a city-ship license by Paris.`
+		log "People" "Beelzebub" `A pirate Bactrian you were responsible for destroying may have been captained by the mysterious Beelzebub whom you have thwarted once before. The Bactrian was destroyed, but you received a message afterwards suggesting they may still not be gone for good, if it was even truly them...`
```

Comment:
These logs should also be present after 3B (as 3A and 3B are mutually exclusive).